### PR TITLE
highlight schedule by moving it to another line

### PR DESCRIPTION
**Pull request by dims**
Also drop the ':' at the end of the line

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

/assign @jberkus

### CODE REVIEW

File: elections/steering/2022/election_desc.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ As is now customary, this fall is [Steering Committee](https://github.com/kubern
 
 Instructions on using Elekto can be found [in its docs site](https://elekto.dev/docs/voting/)
 
-If you’d like to vote or run for a seat, all details and next steps are outlined in the [election process doc](https://git.k8s.io/steering/elections.md) and this application. The application will be the single source of truth of information for this cycle. It will be updated live as new bios of candidates get committed. Please pay attention to the [scheduled dates](https://github.com/kubernetes/community/tree/master/elections/steering/2022#schedule):
+If you’d like to vote or run for a seat, all details and next steps are outlined in the [election process doc](https://git.k8s.io/steering/elections.md) and this application. The application will be the single source of truth of information for this cycle. It will be updated live as new bios of candidates get committed.
+
+Please pay attention to the [scheduled dates](https://github.com/kubernetes/community/tree/master/elections/steering/2022#schedule)
```
Comment (jberkus) [inline elections/steering/2022/election_desc.md:9]:
```suggestion
Please pay attention to the [scheduled dates](https://github.com/kubernetes/community/tree/master/elections/steering/2022#schedule).
```

**Comment by jberkus** [comment]
Edited, needs a period.

**Comment by dims** [comment]
fixed @jberkus

**Pull request merged**